stake-pool/cli/src/main.rs

```diff
@@ -1999,6 +2004,15 @@ fn main() {
                 .global(true)
                 .help("Transaction fee payer account [default: cli config keypair]"),
         )
+        .arg(compute_unit_price_arg().validator(is_parsable::<u64>))
+        .arg(
+            Arg::with_name(COMPUTE_UNIT_LIMIT_ARG.name)
+                .long(COMPUTE_UNIT_LIMIT_ARG.long)
+                .takes_value(true)
+                .value_name("COMPUTE-UNIT-LIMIT")
+                .help(COMPUTE_UNIT_LIMIT_ARG.help)
+                .validator(is_parsable::<u32>)
+        )
```

Comment:
these need to be `.global(true)`, they dont show up for subcommands when running the cli

Reply:
Ah yes, thanks! It turns out I misunderstood how `global` worked this whole time